docs/decisions/0007-maplibre.md

```diff
@@ -0,0 +1,67 @@
+# Use MapLibre for Map Visualization
+
+## Context and Problem Statement
+
+Styling a map with a large number of features, in a performant way, is difficult. With 200,000+ block groups, our project needs this performance for a smooth user experience. Our current mapping approach, using OpenLayers, is insufficient for this task. We propose switching to MapLibre as a free and open source way to improve the performance and user experience of our map.
+
+## Decision Drivers
+
+- Free and Open Source
+- Good performance in rendering a large number of features
+- Good performance in zooming and panning
+- Easy map styling
+
+## Considered Options
+
+- MapLibre GL JS 1.15.0
+- Mapbox GL JS 2.3.1
+- Mapbox GL JS 1.13.0
+- OpenLayers 6.4.3
+
+## Decision Outcome
+
+Chosen option: [MapLibre 1.15.0](https://github.com/maplibre/maplibre-gl-js), a free and open-source alternative to Mapbox with good performance and standard map styling cabilities, and continued development lifecycle.
```

Comment:
It's not just an alternative to Mapbox, right? Maybe we can reword this sentence to be more generic?